tests/anopheles_test_data/test_missing_cohorts/ag/v3-config.json

```diff
@@ -5,5 +5,5 @@
     "GENESET_GFF3_PATH": "reference/genome/agamp4/Anopheles-gambiae-PEST_BASEFEATURES_AgamP4.12.gff3.gz",
     "DEFAULT_SPECIES_ANALYSIS": "aim_20220528",
     "DEFAULT_SITE_FILTERS_ANALYSIS": "dt_20200416",
-    "DEFAULT_COHORTS_ANALYSIS": "20220608"
+    "DEFAULT_COHORTS_ANALYSIS": "20230223"
```

Comment:
Sorry if I've forgotten, but why using 20230223 here rather than 20230516 as used elsewhere in this PR?

Reply:
This one was a mistake, sorry. All updated.